TrafficCapture/testingEnvironment/src/main/docker/nginx/Dockerfile

```diff
@@ -43,8 +43,12 @@ RUN NGINX_ARGS=$(nginx -V 2>&1 | sed -n -e 's/^.*arguments: //p') \
 
 FROM nginx:${NGINX_VERSION}
 
-RUN apt-get update
-RUN apt-get install -y vim
+# Switch to noninteractive mode to avoid interactive prompts during package installation
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends vim \
```

Comment:
Is this to install vim?  What does the process need vim for?  Please leave a comment, especially if it's just nice to have for somebody exec-ing a shell within the container.